lib/initialize.ts

```diff
@@ -2,21 +2,29 @@ import { ConnectMessage, KnownSDK } from './types'
 import connect, { Channel } from './channel'
 
 export default function createInitializer(
-  currentWindow: Window,
-  apiCreator: (channel: Channel, data: ConnectMessage, window: Window) => KnownSDK
+  currentGlobal: typeof globalThis,
+  apiCreator: (channel: Channel, data: ConnectMessage, currentGlobal: typeof globalThis) => KnownSDK
 ) {
+  if (
+    typeof currentGlobal.window === 'undefined' ||
+    typeof currentGlobal.document === 'undefined'
+  ) {
+    // make `init` a noop if window or document is not available
+    return () => {}
```

Comment:
Shall we maybe print a warning to the console in this case?

Reply:
I thought about it, but we'd print for every render when using SSR. That would be quite spammy for an expected behavior